packages/frontend/src/features/filters/hooks/useFormDelay.tsx

```diff
@@ -0,0 +1,20 @@
+import {useEffect} from "react"
+import {Formik} from "formik"
+import _ from "lodash"
+
+export const useFormDelay = (formik: any, phrase: string) => {
```

Comment:
any nie ma obowiazku miec metody handleChange

Reply:
Jak nie znalazłeś typu to zrób interface opisujacy jakie metody / wlasciwosci bedziesz używać z tego obiektu. Dzieki temu ts cie poinformuje gdy przekażesz cos co nie ma metod bez których ta funkcja się wywali